SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -257,24 +260,21 @@ void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel,
 {
 	if (OwningWorld != GetWorld()
 		|| !OwningWorld->IsServer()
+		|| SpatialConnectionManager == nullptr
```

Comment:
I found that I was hitting the old check(SpatialConnectionManager != nullptr);
I talked with Miron about this. He preposed the following changes. 

The issue seems to be OwningWorld->IsServer() is a rubbish check and returns true on clients. 

It is now possible for SpatialConnectionManager  to be nullptr so we return early if it is. 
Miron also asked that we remove CachedLevelsForNetworkIntialize.Add(LoadedLevel);

Reply:
Turns out, maybe we can't remove the CachedLevelsForNetworkIntialize, as I think Jen's PR may break.